core/test/unit/lib/image/image-size_spec.js

```diff
@@ -99,6 +99,40 @@ describe('lib/image: image size', function () {
             }).catch(done);
         });
 
+        it('[success] should return image dimensions when no image extension given', function (done) {
```

Comment:
The mocked test doesn't test that the extra user-agent was added. If i remove the agent, the test still passes. I would suggest to assert that two user agents must be present to fix this edge case?

Reply:
Added a test to assert the newly added user-agent, according to https://github.com/node-nock/nock#request-headers-matching

Reply:
perfecto 👍